File: src/main/g8/$service__packaged$/src/main/$java__packaged$/$servicePackage__packaged$/hello/HelloServiceImpl.java
package com.github.dapeng.hello;

import com.github.dapeng.core.SoaException;
import com.github.dapeng.hello.domain.Hello;
import com.github.dapeng.hello.service.HelloService;

public class HelloServiceImpl implements HelloService{
    @Override
    public String sayHello(Hello hello) throws SoaException {
        return "hello: " + hello.toString();
    }
}

File: src/main/g8/$service__packaged$/src/main/$scala__packaged$/$servicePackage__packaged$/scala/hello/HelloServiceImpl.scala
package com.github.dapeng.scala.hello

import com.github.dapeng.hello.scala.domain.Hello
import com.github.dapeng.hello.scala.service.HelloService

import com.github.dapeng.hello.scala.service.HelloService


class HelloServiceImpl extends HelloService {

  /**
    *
    **/
  override def sayHello(hello: Hello): String = {
    s" hello: \${hello.content} "
  }
}
